master/buildbot/test/unit/test_www_auth.py

```diff
@@ -113,7 +113,7 @@ def setUp(self):
 
     @defer.inlineCallbacks
     def test_maybeAutoLogin(self):
-        self.request.input_headers[b'HDR'] = b'rachel@foo.com'
+        self.request.input_headers[b'HDR'] = 'rachel@foo.com'
```

Comment:
I don't think this change is correct, headers are dict of bytes indexed by bytes

Reply:
@rodrigc what do you think? headers are dict of bytes of bytes, or dict of bytes of strings?

Reply:
@tardyp I think all the changes to `test_www_auth.py` should be reverted.